generic_permissions/visibilities.py

```diff
@@ -60,6 +60,33 @@ def get_attribute(self, instance):
         return queryset
 
 
+class VisibilitySerializerMixin:
```

Comment:
Please add a docstring, I think it's important people understand this!

Reply:
Done